ac_dc/filters.py

```diff
@@ -0,0 +1,99 @@
+from .languages_id import langs_id
+
+
+def lower_strip_sentence(sentence):
+    sent = sentence.lower().strip()
+    return sent
+
+
+def get_words_from_sentence(sentence, strip_characters):
+    """Get lower case words from a sentence."""
+    sent = lower_strip_sentence(sentence)
+    words = [word.strip(strip_characters) for word in sent.split(" ")]
```

Comment:
Splitting on spaces will not work for some languages. We might need to switch to a more proper language based tokenization à-la spacy.

Reply:
Also, one might want to add memoization on this method since it will be used several time. An alternative would be to only tokenize once and allow a richer token based representation of the sentence for later processing.